automated/linux/armnn-benchmarks/armnn-mlperf.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+dhclient
+apt-get install -y ntp
+apt-get --allow-releaseinfo-change update
+wget https://snapshots.linaro.org/componenets/armnn/latest-master/armnn.tar.xz
+tar xf armnn.tar.xz
+cd home/buildslave/workspace/armnn-ci-build
+export BASEDIR=`pwd`
+cd $BASEDIR/armnn/build
+ln -s $BASEDIR/protobuf-host/lib/libprotobuf.so.15.0.0 ./libprotobuf.so.15
+export LD_LIBRARY_PATH=`pwd`
+chmod a+x UnitTests
+df -h
+apt-get install -y git wget curl libz-dev
```

Comment:
please use install_deps from sh-test-lib https://github.com/Linaro/test-definitions/blob/master/automated/lib/sh-test-lib#L221

Reply:
Could you double check that I've implemented install_deps properly in the latest commit?